server/cache.go

```diff
@@ -336,6 +348,18 @@ func (c *clusterInfo) putStoreLocked(store *storeInfo) error {
 	return nil
 }
 
+func (c *clusterInfo) blockStore(storeID uint64) {
+	c.Lock()
+	defer c.Unlock()
```

Comment:
As I said, if the logic is very simple, don't use defer, like here. Use defer when we have to use it.